src/cephadm/cephadm

```diff
@@ -5172,6 +5172,12 @@ def prepare_ssh(
             logger.info('Deploying %s service with default placement...' % t)
             cli(['orch', 'apply', t])
 
+    if not ctx.skip_centralized_logging:
+        for t in ['loki', 'promtail']:
+            logger.info('Deploying %s service with default placement...' % t)
+            cli(['orch', 'apply', t])
+
+
```

Comment:
```suggestion

```
```
cephadm:5182:1: E303 too many blank lines (3)
1     E303 too many blank lines (3)
```
@hookak for flake8 in order to get make check to pass. It seems to want only 2 blank line between the last line here and the next function definition

Reply:
Thanks for comment!